celsos_bookSearch/celsos_bookSearch/Model/Book.swift

```diff
@@ -0,0 +1,22 @@
+//
+//  Book.swift
+//  celsos_bookSearch
+//
+//  Created by Celso Junio Simões de Oliveira Santos on 21/05/21.
+//
+
+import Foundation
+
+public struct Book: Decodable, Encodable {
+    var artworkUrl60:String?
```

Comment:
Question: SwiftLint should be warning me about this typos right? I've installed it with Pods, but it seem that He does not working well in my mac, since it didn't show any warnings.

Reply:
Changes done;
Just waiting to an answer to "Resolve this conversation".